tests/Doctrine/Tests/ORM/Tools/Console/Command/SchemaTool/AbstractCommandTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Tools\Console\Command\SchemaTool;
+
+use Doctrine\DBAL\Platforms\SqlitePlatform;
+use Doctrine\ORM\ORMSetup;
+use Doctrine\ORM\Tools\Console\Command\SchemaTool\AbstractCommand;
+use Doctrine\ORM\Tools\Console\EntityManagerProvider\SingleManagerProvider;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+abstract class AbstractCommandTest extends OrmFunctionalTestCase
+{
+    /**
+     * @param class-string<AbstractCommand> $commandClass
+     */
+    protected function getCommandTester(string $commandClass): CommandTester
+    {
+        $entityManager = $this->getEntityManager(null, ORMSetup::createDefaultAnnotationDriver([
+            __DIR__ . '/Models',
+        ]));
+
+        if (! $entityManager->getConnection()->getDatabasePlatform() instanceof SqlitePlatform) {
+            self::markTestSkipped('We are testing the symfony/console integration');
+        }
+
+        return new CommandTester(new $commandClass(
+            new SingleManagerProvider($entityManager)
+        ));
```

Comment:
I've taken inspiration from https://github.com/doctrine/orm/blob/ab4844b82af9b18cdff20a4d14dc1aeb4ddd08da/tests/Doctrine/Tests/ORM/Tools/Console/Command/ValidateSchemaCommandTest.php#L36-L40 , but I didn't see the point of creating a Symfony application.